website/src/docs/core.md

```diff
@@ -673,14 +669,18 @@ uppy.getPlugin('Dashboard').setOptions({
 })
 ```
 
-### `uppy.reset()`
+### `uppy.reset()` (alias `uppy.cancelAll()`)
 
 Stop all uploads in progress and clear file selection, set progress to 0. More or less, it returns things to the way they were before any user input.
 
-### `uppy.close()`
+### `uppy.close({ reason = 'user' })`
```

Comment:
Shouldn't this also be added for `uppy.cancelAll`?

Reply:
good catch